testing-unit-common/build.gradle

```diff
@@ -38,11 +38,10 @@ android {
 
 dependencies {
     api project(':core')
-    api libs.androidx.test.rules
 
-    implementation libs.androidx.lifecycle.java8
     implementation libs.androidx.lifecycle.runtime
+    implementation libs.kotlin.test
     implementation libs.compose.ui.ui
-    implementation libs.junit
-    implementation libs.kotlin.coroutines.test
+    implementation libs.rxjava2
```

Comment:
Why do you need RxJava here?

Reply:
Please check `NodeViewStub`

Reply:
I think in a follow up PR we should remove the RxJava dependancy. we could make a `testing-rxjava` module potentially

Reply:
`NodeViewStub` was intentionally put into sandbox app to not leak View implementation details (both technical – RxJava and conceptual – Unidirectional flow) into the testing module. If you move it back we can easily remove RxJava dependency from the module.

Reply:
Fair enough. Already removed it